contrib/colab/mnist-training-xrt-1-15.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "[PT Devcon 2019] PyTorch/TPU MNIST",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "TPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FuwrbN1PHChL",
+        "colab_type": "text"
+      },
+      "source": [
+        "## PyTorch/TPU MNIST Demo\n",
+        "\n",
+        "This colab example corresponds to the implementation under [test_train_mnist.py](https://github.com/pytorch/xla/blob/master/test/test_train_mnist.py) and is TF/XRT 1.15 compatible."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tGGfJA4ACw07",
+        "colab_type": "text"
+      },
+      "source": [
+        "### (RUNME) Install Colab TPU compatible PyTorch/TPU wheels and dependencies\n",
+        "This may take up to ~2 minutes\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "un5d9etDG7tO",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "DIST_BUCKET=\"gs://tpu-pytorch/wheels\"\n",
+        "TORCH_WHEEL=\"torch-nightly-cp36-cp36m-linux_x86_64.whl\"\n",
+        "TORCH_XLA_WHEEL=\"torch_xla-nightly-cp36-cp36m-linux_x86_64.whl\"\n",
```

Comment:
If you've tested this with nightly, then this is probably fine. Otherwise we might want to use 1.15 wheel?

Reply:
You're absolutely right! Yes we should be using the frozen xrt 1.15 wheel. I've updated and re-tested all notebooks on Colab sandbox. Thanks for the eye for detail!